### PR TITLE
Add README for MQ-interceptor

### PR DESCRIPTION
**Pull request by kjetilkl**

### CODE REVIEW

File: services/mq-interceptor/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+MQ-interceptor is a proxy service that transfers messages between two AMQP servers.
+
+In the [setup proposed by EGA](https://localega.readthedocs.io/en/latest/amqp.html#connection-to-central-ega), messages are automatically passed between the Central EGA message server 
+and a Local EGA (federated) message server via [federated queues](https://www.rabbitmq.com/docs/federation) or with the use of [shovels](https://www.rabbitmq.com/docs/shovel). 
+However, in some environments, the local message server may not be allowed to connect directly to the central message server for security reasons, 
+and it must therefore rely on a trusted proxy service to pass messages between the two MQ instances.
+Another issue is that the credentials used to authenticate users against the Central EGA server may be different from those used on the Local EGA server.
+
+The MQ-interceptor solves both of these problem by performing the following tasks:
```
Comment (Parisa68) [inline services/mq-interceptor/README.md:28]: problems

Comment (kjetilkl) [reply services/mq-interceptor/README.md]: Fixed!

**Review by Parisa68 — APPROVED** [review]

File: services/mq-interceptor/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+| LEGA_MQ_QUEUE | The name of the queue in the Local MQ server that messages coming from Central MQ should be forwarded to. This value is used as the routing key. Suggested value: "files" |
+| POSTGRES_CONNECTION | A connection string for the PostgreSQL database containing the user ID mappings.<br>`postgres://[username]:[password]@[host]:5432/[database]` | 
+| VERIFY_CERT | If this is set to FALSE, the client will not verify the servers’ certificate chain and host name and will thus accept any certificate presented by the server and any host name in that certificate. This should only be set to FALSE for testing purposes. |
+
+None of these variables have default fallback values, and MQ-interceptor will usually fail to start if they are not set explicitly. The only variable that is not required is VERIFY_CERT, and the MQ-interceptor will always verify certificates unless this variable is explicitly set to "false".
+
+## Build and execute
+The MQ-interceptor is written in the [GO language](https://go.dev/) and requires the [GO compiler](https://go.dev/doc/install) to build.
```
Comment (kjellp) [inline services/mq-interceptor/README.md:79]:
```suggestion
The MQ-interceptor is written in the [Go language](https://go.dev/) and requires the [Go compiler](https://go.dev/doc/install) to build.
```

**Review by kjellp — CHANGES_REQUESTED** [review]
Looks good, please fix recommended capitalization fix before merging,

File: services/mq-interceptor/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+| CEGA_MQ_EXCHANGE | The name of the exchange to post messages to on the Central MQ server. Suggested value: "localega.v1" |
+| CEGA_MQ_QUEUE | The name of the queue to read messages from on the Central MQ server. Suggested value: "v1.files" |
+| LEGA_MQ_CONNECTION |  A connection string for connecting to the Local EGA message server.<br>`amqps://<user>:<password>@<host>:<port>/<vhost>[?parameters]` |
+| LEGA_MQ_EXCHANGE | The name of the exchange to post messages to on the Local MQ server. Suggested value: "sda" |
+| LEGA_MQ_QUEUE | The name of the queue in the Local MQ server that messages coming from Central MQ should be forwarded to. This value is used as the routing key. Suggested value: "files" |
+| POSTGRES_CONNECTION | A connection string for the PostgreSQL database containing the user ID mappings.<br>`postgres://[username]:[password]@[host]:5432/[database]` | 
+| VERIFY_CERT | If this is set to FALSE, the client will not verify the servers’ certificate chain and host name and will thus accept any certificate presented by the server and any host name in that certificate. This should only be set to FALSE for testing purposes. |
```
Comment (pavelvazquez) [inline services/mq-interceptor/README.md:74]:
```suggestion
| VERIFY_CERT | If this is set to FALSE, the client will not verify the server’s certificate chain and host name and will thus accept any certificate presented by the server and any host name in that certificate. This should only be set to FALSE for testing purposes. |
```

Comment (pavelvazquez) [reply services/mq-interceptor/README.md:74]: I believe in a single server

**Review by pavelvazquez — DISMISSED** [review]

File: services/mq-interceptor/README.md (hunk 3 above)
Comment (kjetilkl) [reply services/mq-interceptor/README.md:74]: There are two servers

**Review by pavelvazquez — APPROVED** [review]

**Review by kjellp — APPROVED** [review]

**Pull request merged**